build.gradle

```diff
@@ -34,6 +34,7 @@ allprojects {
 subprojects {
     apply plugin: 'java-library'
     apply plugin: 'jacoco'
+    apply plugin: 'maven-publish'
```

Comment:
This allows us to run `gradlew publishToMavenLocal` to publish locally for testing in other projects